tests/test_table.py

```diff
@@ -220,6 +220,9 @@ def test_pick_files(files, table):
     table = table.pick_files(files, inplace=False)
     if callable(files):
         files = table.files[table.files.to_series().apply(files)]
+        seen = set()
+        seen_add = seen.add
+        files = [x for x in files if not (x in seen or seen_add(x))]
```

Comment:
What is this doing?

Reply:
Removes duplicates. We need this now since otherwise `audformat.create_filewise_index()` in the assertion will fail.

Reply:
And we cannot use `set()` here as we need to keep the ordering.